tasmota/xdrv_27_shutter.ino

```diff
@@ -230,7 +214,7 @@ void ShutterInit(void)
       dtostrfd((float)Shutter.open_time[i] / 10 , 1, shutter_open_chr);
       char shutter_close_chr[10];
       dtostrfd((float)Shutter.close_time[i] / 10, 1, shutter_close_chr);
-      AddLog_P2(LOG_LEVEL_INFO, PSTR("SHT: Shutter %d (Relay:%d): Init. Pos: %d [%d %%], Open Vel.: 100 Close Vel.: %d , Max Way: %d, Opentime %s [s], Closetime %s [s], CoeffCalc: c0: %d, c1 %d, c2: %d, c3: %d, c4: %d, binmask %d, is inverted %d, shuttermode %d,motordelay %d"),
+      AddLog_P2(LOG_LEVEL_INFO, PSTR("SHT: Shutter %d (Relay:%d): Init. Pos: %d [%d %%], Open Vel.: 100 Close Vel.: %d , Max Way: %d, Opentime %s [s], Closetime %s [s], CoedffCalc: c0: %d, c1 %d, c2: %d, c3: %d, c4: %d, binmask %d, is inverted %d, shuttermode %d,motordelay %d"),
```

Comment:
I think you reintroduce e typo here

Reply:
> secure manually changing relays will not overrun end positions.   

Regarding this point it's bad news for me.
- My shutters have clutches so that they can only go reverse when they reach end of course, they are not concerned by overrun.
- But when position=0% and the shutter is not physically closed, it's really a pain to use the buttons to reach the physical closed position, a 0.5s pulse of overrun would be welcome, for friends/kids/wife that only have the manual wall switches to adjust the position.

Reply:
Please check the introduces shuttermotordelay a few weeks ago. This was actually introduced to help situations like yours where the shutters do not start immediately and therefore create a shift. You can define it from 0.00 to 12.75 seconds. Stepping 0.05sec

Reply:
It seems to solve my issue. I guess I'll further have to subtract that value in ShutterOpenDuration and ShutterCloseDuration but it'll be easy, I already recorded all the backlog commands in a file.
Thanks